src/main/java/taskat4/Human.java

```diff
@@ -0,0 +1,91 @@
+package taskat4;
+
+import java.io.IOException;
+
+public abstract class Human {
+
+    private final boolean gender;
+    private String name;
+    private String surname;
+    private float height;
+    private float weight;
+
+    public Human(boolean gender, String name, String surname, float height, float weight) {
+        this.gender = gender;
+        this.name = name;
+        this.surname = surname;
+        this.height = height;
+        this.weight = weight;
+    }
+
+    public String toString(boolean bo) {
```

Comment:
what sense of this method? Do you understand that this way you do not override toString(), but overload it?

Reply:
Now I`ve got it